app/src/main/java/com/aaronjwood/portauthority/activity/MainActivity.java

```diff
@@ -289,8 +289,7 @@ public void run() {
      */
     private void getInternalIp() {
         int netmask = this.wifi.getInternalWifiSubnet();
-        int count = Integer.bitCount(netmask);
-        String InternalIpWithSubnet = this.wifi.getInternalWifiIpAddress() + "/" + Integer.toString(count);
+        String InternalIpWithSubnet = this.wifi.getInternalWifiIpAddress() + "/" + Integer.toString(netmask);
```

Comment:
I'd like to support returning the netmask as a integer since it's much easier to do networking operations on it versus string manipulation.  I could overload another method to return a string to support both return values.